JSDemos/Demos/DataGrid/RecordPaging/description.md

```diff
@@ -1 +1,12 @@
-The **DataGrid** widget can automatically split its content across multiple pages and allow end users to&nbsp;navigate records using its built-in pager. This demo illustrates how you can change pager settings by&nbsp;using both the **paging** and **pager** configuration options.
\ No newline at end of file
+The **DataGrid** automatically splits records across multiple pages. This optimizes performance as the **DataGrid** loads and renders only the records on the current page. Users can scroll the pages or navigate between them with the pager. This demo illustrates the second case.
+
+The pager is configured in the [pager](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/pager/) object and consists of the following elements:
+
+- **Page navigator**        
+Used to switch between pages.
+
+- **Page size selector**        
+Used to change the page size. To display this element, enable the [showPageSizeSelector](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/pager/#showPageSizeSelector) option. You can also define the [allowedPageSizes](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/pager/#allowedPageSizes) and specify the initial [pageSize](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/paging/#pageSize) in the [paging](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/paging/) object.
+
+- **Page information**      
+Displays the current page number and total record count. To display page information, enable the [showInfo](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/pager/#showInfo) option. You can also customize the [infoText](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/pager/#infoText).
```

Comment:
For now, it seems unclear what is infoText and why we should customize it. I'm not really sure, but maybe 'You can also use the [pageSize]() option to customize row and page counters'?

Reply:
How about this?
```suggestion
Displays the current page number and total record count. To display page information, enable the [showInfo](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/pager/#showInfo) option. You can also use the [infoText](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/pager/#infoText) option to customize the information text string.
```